static/app/utils/discover/urls.tsx

```diff
@@ -41,37 +41,43 @@ export function eventDetailsRoute({
  * TODO Abdullah Khan: Add link to new trace view doc explaining why we route to the traceview.
  */
 export function generateLinkToEventInTraceView({
```

Comment:
Changed how we pass params to this function. Previously we expected `table data row`, but links to event can exist outside of a table so just passing the fields as de-structured params now.